### PR TITLE
Update course listing table

### PR DESCRIPTION
**Pull request by defong**
### Context
Meeting design

same or better then
image

https://manage-courses-prototype.herokuapp.com/history/preparing-for-locations#courses

### Changes proposed in this pull request
Making it work

### Guidance to review
screencapture-localhost-44364-organisation-2dg-courses-2019-02-07-14_06_40

### CODE REVIEW

**Comment by defong** [comment]
[Update courses table](https://trello.com/c/HKX8RZcA/936-update-courses-table)

File: ManageCoursesUi.Tests/Controllers/OrganisationControllerTests.cs [hunk 1]
```diff
@@ -1,633 +1,632 @@
...
-
-            var result = await controller.Show(providerCode);
-
-            var viewResult = result as ViewResult;
-
-            Assert.IsNotNull(viewResult);
-            var model = viewResult.ViewData.Model as CourseListViewModel;
-
-            Assert.NotNull(model);
-            Assert.AreEqual(1, model.Providers.Count);
```
Comment (timabell) [inline ManageCoursesUi.Tests/Controllers/OrganisationControllerTests.cs]: separate the reformatting from the actual change

File: src/ui/Controllers/CourseController.cs [hunk 2]
```diff
@@ -522,5 +522,43 @@ private CourseRouteDataViewModel GetCourseRouteDataViewModel(string providerCode
                 CourseCode = courseCode
             };
         }
+        [Route("{providerCode}/courses")]
+        public async Task<IActionResult> Index(string providerCode)
+        {
+            // NEW!!!!
```
Comment (timabell) [inline src/ui/Controllers/CourseController.cs]: eh?

**Comment by timabell** [comment]
needs rebasing

**Comment by timabell** [comment]
rebased to give cleaner history

**Pull request merged**